sdk/core/core-amqp/test/requestResponse.spec.ts

```diff
@@ -399,4 +409,117 @@ describe("RequestResponseLink", function() {
       );
     }
   });
+
+  describe("sendRequest clears timeout", () => {
+    const _global = getGlobal();
+    const originalClearTimeout = clearTimeout;
+    afterEach(() => {
+      _global.clearTimeout = originalClearTimeout;
+    });
+
+    it("sendRequest clears timeout after error message", async function() {
+      let wasClearTimeoutCalled = false;
+      _global.clearTimeout = (tid) => {
+        wasClearTimeoutCalled = true;
```

Comment:
Unlikely but...how do you know that the `tid` cleared is _your_ `tid`?

Reply:
I could change this to a count to better future-proof. The reason I know now because "there can only be one".

Reply:
I did a slightly goofy thing there I changed the code to setTimeout but with a named function so I could check it in my hooked setTimeout.

Now that is with the caveat that I somehow caused bad interaction with the entire test suite in that test....